src/app/components/Icons/FacebookIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export default function FacebookIcon(): JSX.Element {
+  return (
+    <svg width="43.812" height="43.812" viewBox="0 0 43.812 43.812">
+      <defs>
+        <linearGradient
+          id="linear-gradient"
+          x1="0.5"
+          x2="0.5"
+          y2="1"
+          gradientUnits="objectBoundingBox"
+        >
+          <stop offset="0" stop-color="var(--primary-color)" />
+          <stop offset="1" stop-color="#ffa90a" />
```

Comment:
See https://github.com/lmachens/photoplay/blob/main/src/app/globals.css#L14-L15
```suggestion
          <stop offset="0" stop-color="var(--primary-gradient-from)" />
          <stop offset="1" stop-color="var(--primary-gradient-to)" />
```

Reply:
thx